### ISSUE TITLE
Suggestion: Custom Tab in Scenario Selector

### ISSUE THREAD

**Issue by harrysbird** [OpenRCT2/OpenRCT2#5235]
I Have Suggestion in OpenRCT2

I Would to like have Custom Tab in Scenario Selector because I love to make my park so I want to make custom tab so I don't put my park in other park tab, Great idea to add this feature.
![suggestion idea](https://cloud.githubusercontent.com/assets/3393747/23088984/2822686a-f537-11e6-9dc9-6318cf1c573e.jpg)

**Comment by Liampie** [comment]
I think it's a good idea to add a sixth tab for your own scenarios, but if you give every user the option to customize scenarios will be incompatible to other people's games. That's just a hassle.

**Comment by grimpunch** [comment]
I agree as far as much as a "My Scenario's" tab
As that means in the future we could also have a "Community Scenarios" tab in future if we sort out some kind of sharing mechanism

**Comment by Gymnasiast** [comment]
IMO there should be support for creating extra tabs in general. For example, a scenario pack could add one.

**Comment by harrysbird** [comment]
Community Scenarios tab will take lot of work for that's

**Comment by marvk** [comment]
I think every sub-folder in the `OpenRCT2/scenario` folder should be it's own tab, thus making it easily possible to distinguish between different map packs if you have the scenarios in their separate folders.

**Comment by IntelOrca** [comment]
I'd say a tree view would be better for that.

**Comment by marvk** [comment]
Good idea, you could combine them so you can select a folder in a tree view and show all scenarios in that specific folder in the view that already exists, thus not mixing scenarios and folders in one view while also not flooding the tab bar with a bunch of tabs if you have a lot of subfolders.

**Comment by Patrik356b** [comment]
Tree View would be excellent i think

**Comment by player49256** [comment]
I'll paste my comment from the duplicate issue I opened here (the one AaronVanGeffen referenced).
-------------------------------------------
It would be cool if you could create your own scenario groups/tabs for your own missions. This could allow players to create scenario packs and group them nicely within the game, allowing people to play them and unlock like you do with the built in scenarios today. Also include the feature to unlock bonus parks (like Mega Park in Classic).

Not sure about best way to implement but a simple way would be a index file with name and which scenarios to include, including order of the scenarios. They could be placed in subfolder within "scenario" along with the actual sc4/sc6 files. They could also be placed in a seperate scgroup folder.

The game then reads those files on launch and includes them in the selectable tabs.

An option would be to in the Ingame Options check/uncheck group to prevent overflow of tabs, instead of manually deleting/renaming files.

Maybe in the future even an ingame editor for scenario groups?
